projects/user-telemetry-client/tsconfig-build.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Added to keep tests our of our `dist` folder?

Reply:
Whilst also adding the types to out tests, I would be interested to hear if there are better ways.